File: python-learning/object.py
class Car:

    type = "Fiat"
    model = "500"
    color = "white"

#method
    def start(self):
        print("Ini method start")

    def drive(self):
        print("ini method drive")

    def brake(self):
        print("Ini method brake")

    def stop(self):
        print("Ini method stop")

# Membuat objek mobil
car = Car()
print(car.type)
print(car.color)

# Memanggil method 
car.start()
car.drive()
car.brake()
car.stop()

File: javascript-learning/part-2/object.js
var car = {
    //properti
    type: "Fiat",
    mode: "500",
    color: "white",
    date:  {
        year :2023,
        month : "December",
        date: 25,
    },

    data:[
        1,2,3,4,5
    ],

    //method
    start: function(){
        console.log("ini method start");
    },
    drive: function(){
        console.log("ini method drive")
    },
    brake: function(){
        console.log("ini method brake")
    },
    stop: function(){
        console.log("ini method stop")
    }

};

car.start()
console.log(car.data)